Lib9c/Model/State/GameConfigState.cs

```diff
@@ -448,6 +457,20 @@ public override IValue Serialize()
                     RequireCharacterLevel_ConsumableSlot5.Serialize());
             }
 
+            if (RuneSkillSlotCrystalUnlockCost > 0)
```

Comment:
Is it ok to other external service to create rune action manually?

Reply:
I didn't understand the comment. could you please explain?